run.py

```diff
@@ -23,6 +23,7 @@
                         help='Goal of the classifier',
                         choices=['bug', 'regression', 'tracking', 'qaneeded', 'uplift', 'component', 'devdocneeded', 'defectfeaturetask'],
                         default='bug')
+    parser.add_argument('--ngrams', default='1')
```

Comment:
Not all classifiers support this, so I'm not a fan of adding the option in run.py.